.github/workflows/rust.yml

```diff
@@ -159,3 +159,13 @@ jobs:
 
     - name: Build
       run: cargo xtask build
+
+  build_feature_permutations:
```

Comment:
Unfortunately, this will increase our CI times :/
But for what it's worth, this is a good thing to have, I think

Reply:
It might not increase the time actually since the jobs are run in parallel. The times vary from one run to another, but for example in https://github.com/rust-osdev/uefi-rs/actions/runs/3557618653/ the new job took 3m26s, but the aarch64 job took 3m39s, the lint job took 3m40s, and the Windows job took 3m50s. So CI times seem likely to remain about the same as before.

Reply:
Okay, then forget what I've said. Probably, it is not a big problem then :)